src/openms/include/OpenMS/FORMAT/FASTAFile.h

```diff
@@ -38,187 +38,175 @@
+            }
+
+            bool operator==(const FASTAEntry& rhs) const
+            {
+                return identifier == rhs.identifier
+                       && description == rhs.description
+                       && sequence == rhs.sequence;
+            }
+
+            bool headerMatches(const FASTAEntry& rhs) const
+            {
+                return identifier == rhs.identifier &&
+                       description == rhs.description;
+            }
+
+            bool sequenceMatches(const FASTAEntry& rhs) const
+            {
+                return sequence == rhs.sequence;
+            }
+        };
+
+        /// Default constructor
+        FASTAFile();
+
+        /// Destructor
+        virtual ~FASTAFile();
+
+        /**
+          @brief Prepares a FASTA file given by 'filename' for streamed reading using readNext().
+          @exception Exception::FileNotFound is thrown if the file does not exists.
+          @exception Exception::ParseError is thrown if the file does not suit to the standard.
+        */
+        void readStart(const String& filename);
+
+        /**
+        @brief Reads the next FASTA entry from file.
+        If you want to read all entries in one go, use load().
+        @return true if entry was read; false if eof was reached
+        @exception Exception::FileNotFound is thrown if the file does not exists.
+        @exception Exception::ParseError is thrown if the file does not suit to the standard.
+        */
+        bool readNext(FASTAEntry& protein);
+
+        /// current stream position
+        std::streampos position();
+
+        /// is stream at EOF?
+        bool atEnd();
+
+        /// seek stream to @p pos
+        bool setPosition(const std::streampos& pos);
+
+        /**
+        @brief Prepares a FASTA file given by 'filename' for streamed writing using writeNext().
+        @exception Exception::UnableToCreateFile is thrown if the process is not able to write to the file (disk full?).
+        */
+        void writeStart(const String& filename);
+
+        /**
+        @brief Stores the data given by @p protein. Call writeStart() once before calling writeNext().
+        Call writeEnd() when done to close the file!
+        @exception Exception::UnableToCreateFile is thrown if the process is not able to write the file.
+        */
+        void writeNext(const FASTAEntry& protein);
+
+        /**
+        @brief Closes the file (flush). Called implicitly when FASTAFile object does out of scope.
+        */
+        void writeEnd();
+
+
+        /**
+          @brief loads a FASTA file given by 'filename' and stores the information in 'data'
+          This uses more RAM than readStart() and readNext().
+          @exception Exception::FileNotFound is thrown if the file does not exists.
+          @exception Exception::ParseError is thrown if the file does not suit to the standard.
+        */
+        void static load(const String& filename, std::vector<FASTAEntry>& data);
```

Comment:
something has gone wrong during merging/conflict resolving.

Please undo your changes to `load()` and `store()` (incl. documentation).